### ISSUE TITLE
Service Inventory release follow-up items

### ISSUE THREAD

**Issue by Stephen-ONeil** [TBS-EACPD/infobase#1326]
From [this comment](https://github.com/TBS-EACPD/infobase/pull/1299#issuecomment-984716377)

Things to follow up on:
  - More urgent:
     - [ ] [this](https://app.circleci.com/pipelines/github/TBS-EACPD/infobase/5677/workflows/136631b0-6572-48af-80f1-debe6f105a70/jobs/42266) job should NOT have passed, a11y versions of the dept level service route (at a minimum) were throwing errors!
      - testing on #1325
    - [ ] in calculating percentage of service available online end-to-end, should NAs be excluded from the denominator?
    - [ ] these labels are a mess in French
![image](https://user-images.githubusercontent.com/11301919/144552522-46ec6d4f-86f2-41c3-8255-7426fe0bcba8.png)
  - Less urgent:
    - [ ] are all years in the service data actually fiscal years? If not, want a double check on which are which
      - [ ] either way, should be numbers, not strings, for consistency with other API years
    - [ ] going to want to refactor all the models along the same lines as the covid models for multiple years of service data, eventually
    - [ ] why no CRSO level services infographics? 
    - [ ] nulls vs 0s in data, are those meaningful and where? Those are a big source of bugs client side, if necessary would prefer to default them to 0 and, if necessary 
      - orrrr revisit the question of making the `DisplayTable` "understand" nulls be default...
      - didn't refresh my memory on this too closely, it may be that the default sorting explicitly rejects mixed content columns. A column of ALL empties may be fine? Double check this stuff when coming back to it
    - [ ] what's with the name "Customizable Service Graph"? Weak
    - [ ] are `first_active_year`/`last_active_year` useless?
      - ... thought I could use first_active_year/last_active_year to identify dead services but looks like dead services may have no 2019 submission and, in 2018, have no last_active_year. So last_active_year isn't going to ever be set?
      - all these first_active_years of 2018... can't be right, this must be more like first year reported on, which isn't really worth displaying to users
    - [ ] strange horizontal ticks [here](https://www.tbs-sct.gc.ca/ems-sgd/edb-bdd/index-eng.html#infographic/program/INDSC-BXL01/services/.-.-(panel_key.-.-'application_channels_by_services)), shouldn't all be 5,000,000. Also generally have the potential to get cut off in this panel
![image](https://user-images.githubusercontent.com/11301919/144634881-11744da6-2dca-4b3e-9192-56a5cec014bb.png)

**Comment by Stephen-ONeil** [comment]
@ktw1016 try and work on some of these/answer any of my questions you can. Right now I'm only digging in to the urgent one about cypress tests.

**Comment by ktw1016** [comment]
> in calculating percentage of service available online end-to-end, should NAs be excluded from the denominator?
For online end-to-end calculation, Mike is defining "If all six interaction points are not applicable, then that service is not online end-to-end". So NAs should be treated the same "not enabled" service. (Binary, either enabled or not enabled).

> are all years in the service data actually fiscal years?
> should be numbers, not strings
I'm not exactly sure what "fiscal year" is defined as, but you're right, they should simply be numbers. "Fiscal year" as such 2018-19 just means collection year.

> why no CRSO level services infographics?
Hard to recall but I think I've suggested this to Mike at the beginning and it just kinda carried away. We could bring it up to them

> nulls vs 0s in data, are those meaningful and where? Those are a big source of bugs client side, if necessary would prefer to default them to 0 and, if necessary
This is pretty difficult that won't be easily resolvable because data is mixed and inconsistent.. I know that Mike told me some departments entered 0 as "not available". Something that'll need to be discussed further with OCIO

> what's with the name "Customizable Service Graph"? Weak
Totally agreed and I actually brought it up to Mike a several times to change, where he insisted he wants this name specifically😂

> are first_active_year/last_active_year useless?
>... thought I could use first_active_year/last_active_year to identify dead services but looks like dead services may have no >2019 submission and, in 2018, have no last_active_year. So last_active_year isn't going to ever be set?
This was the exact point I made to Chris. We need a field like `is_active` for each year of service data and determine exactly which years it was available for. Chris didn't want to make this change (or it wasn't a priority). They're pretty useless fields as of now

**Comment by Stephen-ONeil** [comment]
Another one I forgot, is it intentional that the home page services icon isn't the same as the bubble menu version (alt colours aside)?

**Comment by ktw1016** [comment]
No, that wasn't intentional, I was just trying to fit SVG into the card component giving colors, and kinda looks a bit stretched

**Comment by Stephen-ONeil** [comment]
It's a whole different icon.
